svgnative/ports/cg/CGSVGRenderer.cpp

```diff
@@ -75,11 +77,41 @@ void CGSVGTransform::Concat(const Transform& other)
     mTransform = CGAffineTransformConcat(mTransform, static_cast<const CGSVGTransform&>(other).mTransform);
 }
 
+CGSVGImageData::CGSVGImageData(const std::string& base64, ImageEncoding encoding)
+{
+    std::string imageString = base64_decode(base64);
+    auto dataProvider = CGDataProviderCreateWithCFData(CFDataCreate(NULL, (const UInt8*)imageString.data(), imageString.size()));
+    if (encoding == ImageEncoding::kPNG)
+        mImage = CGImageCreateWithPNGDataProvider(dataProvider, NULL, true, kCGRenderingIntentDefault);
+    else if (encoding == ImageEncoding::kJPEG)
+        mImage = CGImageCreateWithJPEGDataProvider(dataProvider, NULL, true, kCGRenderingIntentDefault);
+}
+
+CGSVGImageData::~CGSVGImageData()
+{
+    CGImageRelease(mImage);
```

Comment:
Check for nullptr. Apple's XyzRelease() APIs usually crash when passed a nullptr.